docs/09_Pages/Navigating_Pages/Internal_Page_on_App_Load.md

```diff
@@ -0,0 +1,37 @@
+The default behavior for DrupalGap is to direct to the front page set in the `settings.js` file. This breaks the sharing feature that users are used to on a typical web site. This code may be added to `settings.js` to enable direction to internal pages.
+
+**This method has only been tested in a web app.**
+
+```
+var landingPage = 'dashboard';
```

Comment:
Yes, the regex changes the format from the hash value to what the DrupalGap page navigation is expecting for a node page.